src/uproot/models/RNTuple.py

```diff
@@ -52,6 +51,9 @@ def _renamemeA(chunk, cursor, context):
 
 
 class Model_ROOT_3a3a_Experimental_3a3a_RNTuple(uproot.model.Model):
+    def debug(self):
+        return self.header.field_records, self.header.column_records
```

Comment:
We can leave in `debug` methods if they provide enough value to future maintainers (ourselves). This one, though, doesn't provide much benefit above fetching these two properties manually, so it should be dropped.